.vsts.release.yml

```diff
@@ -52,7 +59,7 @@ extends:
   parameters:
     branch: $[ stageDependencies.Verify_release.Set_variables.outputs['SetReleaseVariables.releaseBranch'] ]
     test: ${{ not(parameters.skipTests) }}
-    sign: true
+    sign: false
```

Comment:
What's the reason of this change?

Reply:
Thanks for noticing, I disabled it temporarily, because the slowness of code signing service on macos